sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/sm_pipeline_with_amazon_forecast.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    region_to_account_id = {
</pre><p>This should be fetched programmatically rather than hard-coded, using the <code>sagemaker.image_uris.retrieve()</code>  function. </p><p>Such as: </p><pre class="ql-syntax" spellcheck="false">import sagemaker
image_uri = sagemaker.image_uris.retrieve(framework='sklearn',region=region,version='0.23-1',image_scope='training')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>